### ISSUE TITLE
Improve rules with comma-separated values

### ISSUE THREAD

**Issue by tfrommen** [cssstats/cssstats#173]
Currently, cssstats does not consolidate two rules that are equal except for whitespace (after commas). If you take a look at Google, you see both `arial,sans-serif`and `arial, san-serif`. These would bestbe treated as one and the same rule. (Just like `'arial',...` and `"arial",...`.)

**Comment by mrmrs** [comment]
Yes this is intentional as we want to highlight where there are inconsistencies in declarations, not just what a computed value will render as.
